packages/rollup-core/src/app/data/data-service.ts

```diff
@@ -42,29 +44,35 @@ export class DefaultDataService implements DataService {
    */
```

Comment:
Idk what a good name for this would be though hmm...

Reply:
Yeah, this is a tough balancing act because don't want it tied to postgres or even a DB, but do want to indicate that "this is the thing you use to CRUD data"